### PR TITLE
Tasks connected to a swarm network will have 1 endpoint on windows RS3.

### PR DESCRIPTION
**Pull request by pradipd**

### CODE REVIEW

**Comment by pradipd** [comment]
@msabansal please review

File: drivers/windows/overlay/ov_endpoint_windows.go [hunk 1]
```diff
@@ -15,12 +19,14 @@ type endpointTable map[string]*endpoint
 const overlayEndpointPrefix = "overlay/endpoint"
 
 type endpoint struct {
```
Comment (pradipd) [inline drivers/windows/overlay/ov_endpoint_windows.go]: Eventually, we should look at converging type endpoint struct and type hnsEnpoint struct in windows pkg so they can be shared by all windows and overlay drivers.  But, I think that is outside the scope of this PR.

File: drivers/windows/overlay/ov_endpoint_windows.go [hunk 2]
```diff
@@ -141,6 +149,31 @@ func (d *driver) CreateEndpoint(nid, eid string, ifInfo driverapi.InterfaceInfo,
 
 	hnsEndpoint.Policies = append(hnsEndpoint.Policies, paPolicy)
 
+	if system.GetOSVersion().Build > 16236 {
```
Comment (pradipd) [inline drivers/windows/overlay/ov_endpoint_windows.go]: Will move this check into hns.

Comment (msabansal) [inline drivers/windows/overlay/ov_endpoint_windows.go]: lgtm

**Comment by msabansal** [comment]
LGTM

**Comment by pradipd** [comment]
@mavenugo @fcrisciani Can we merge this pull request?

**Comment by pradipd** [comment]
@mavenugo @fcrisciani Or review it and give feedback so that we can merge it.

**Comment by fcrisciani** [comment]
LGTM, @pradipd can you confirm that is ready to be merged?

**Comment by pradipd** [comment]
@fcrisciani it is ready to be merged.
